core/fortran-example/advanced/01_parallel_for/README.md

```diff
@@ -0,0 +1,65 @@
+This directory contains two examples of Cabana for Fortran that can be readily adopted for users wanting to build a PIC code or incorporate Cabana kernels into an existing Fortran PIC code.
```

Comment:
This is markdown, so you backquotes (i.e. `) to mark code.

Reply:
@ascheinb to address it in the new branch.